git.yazi/main.lua

```diff
@@ -29,11 +29,8 @@ local function match(line)
 			path = line:sub(4, 4) == '"' and line:sub(5, -2) or line:sub(4)
 			path = WINDOWS and path:gsub("/", "\\") or path
 		end
-		if not path then
-		elseif path:find("[/\\]$") then
-			return p[2] == CODES.ignored and CODES.excluded or p[2], path:sub(1, -2)
-		else
-			return p[2], path
+		if path then
```

Comment:
Please separate this logical change apart from the rest renaming refactor (rename `changed` to `changes`, add or remove new lines, `tostring`s, etc.)